workflow-bot-app/src/test/java/com/symphony/bdk/workflow/engine/camunda/monitoring/converter/VariableDomainConverterTest.java

```diff
@@ -0,0 +1,31 @@
+package com.symphony.bdk.workflow.engine.camunda.monitoring.converter;
+
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.symphony.bdk.workflow.monitoring.repository.domain.VariablesDomain;
+
+import org.assertj.core.util.Maps;
+import org.camunda.bpm.engine.history.HistoricVariableUpdate;
+import org.junit.jupiter.api.Test;
+
+import java.util.Date;
+
+class VariableDomainConverterTest {
+
+  @Test
+  void apply() {
+    HistoricVariableUpdate detail = mock(HistoricVariableUpdate.class);
+    when(detail.getRevision()).thenReturn(1);
+    when(detail.getTime()).thenReturn(new Date());
+    when(detail.getValue()).thenReturn(Maps.newHashMap("key", "value"));
+
+    VariableDomainConverter converter = new VariableDomainConverter();
+    VariablesDomain domain = converter.apply(detail);
+
+    then(domain.getRevision()).isEqualTo(1);
```

Comment:
What's the interest of using then() instead of assertThat()? 

Do you really want to introduce a new assertion method 'then' or keep using assertThat like in the rest of tests?